src/SFA.DAS.EmploymentCheck.Functions/Application/Clients/EmploymentCheck/IEmploymentCheckClient.cs

```diff
@@ -10,6 +12,8 @@ public interface IEmploymentCheckClient
 
         Task<IList<EmploymentCheckCacheRequest>> CreateEmploymentCheckCacheRequests(EmploymentCheckData employmentCheckData);
 
+        Task<IList<EmploymentCheckCacheRequest>> SetEmploymentCheckCacheRequestRelatedRequestsRequestProcessingStatus(Tuple<EmploymentCheckCacheRequest, ProcessingCompletionStatus> employmentCheckCacheRequestAndStatusToSet);
```

Comment:
anything more concise but as descriptive that could be used instead of `SetEmploymentCheckCacheRequestRelatedRequestsRequestProcessingStatus` ?? :)